qa/evil-tests/src/test/java/org/elasticsearch/bootstrap/BootstrapCliParserTests.java

```diff
@@ -55,6 +65,7 @@ public void clearProperties() {
             System.clearProperty(property);
         }
         propertiesToClear.clear();
+        assertEquals("properties leaked", properties, new HashMap<>(System.getProperties()));
```

Comment:
I can do that but I don't wanna leak this into the real tests since they are protected by the security manager?